src/Native/CMakeLists.txt

```diff
@@ -8,6 +8,14 @@ set(RESOURCES)
 include_directories("${CMAKE_BINARY_DIR}/../../")
 
 if(WIN32)
+    # Clobber and reset the default C and CXX flags because
+    # CMake uses /ZI (Edit and Continue) for generating pdbs
+    # which is incompatible with the /guard:cf flag we set below
+    # for security. So we use the default flags set by CMake 
+    # and reset /ZI with /Zi
+    set(CMAKE_C_FLAGS_DEBUG "/MDd /Zi /Ob0 /Od /RTC1 /JMC")
```

Comment:
Where do those flags `/MDd /Zi /Ob0 /Od /RTC1 /JMC` come from? Are they default flags in CMake? Could you please add more details for maintaining them in the future?

Reply:
Yes, those are the default flags in CMake. I will add a bit more detail and submit another PR soon.